flipt-node/README.md

```diff
@@ -17,3 +17,42 @@ npm i @flipt-io/flipt@{version}
 ## Usage
 
 In the [example](./example) directory, there is an example TypeScript program which imports in the flipt client, and uses it appropriately, please refer to that for how to use the client.
+
+### Metrics
+
+There is support for [Datadog RUM](https://docs.datadoghq.com/real_user_monitoring/) through this client. This allows you to track the values of feature flag evaluation and how it relates to active browser sessions.
+
+To start tracking feature flags on Datadog:
+
+```typescript
+import { datadogRum } from '@datadog/browser-rum';
```

Comment:
actually, do we want to include this in our package.json for this client?

Reply:
or not? i dont remember what is best practice/required

Reply:
@markphelps Yeah I think when I initially did this, there seemed like a lot to configure for the RUM client, which is why I left initialization to the host program.

Reply:
But I am fine to do it either way is necessary.

Reply:
it might be good to leave it to the user then, as I can see some users not wanting it installed for them if they arent using it. so ignore this root comment !

Reply:
we may just want to document in the README how to install the datadog rum client though